### PR TITLE
📖  Bump KubeFlex dependency to 0.7.1

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR tracks the creation of KubeFlex release 0.7.1. The 0.7 line includes some CLI improvements, and this PR takes advantage of them to make our instructions and scripts simpler and more reliable.

This PR and #2520 need to be serialized. Once one of them merges, the other will need updating (I will happily do that).

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
/cc @pdettori 
/cc @clubanderson 
@katamyra

File: docs/content/direct/get-started.md [hunk 1]
```diff
@@ -32,11 +32,9 @@ you will first want to remove any related debris. The following
 commands tear down the state established by this recipe.
 
 ```shell
-yq -i 'del(.preferences)' ${KUBECONFIG:-$HOME/.kube/config}
 kind delete cluster --name kubeflex
 kind delete cluster --name cluster1
 kind delete cluster --name cluster2
-kubectl config delete-context kind-kubeflex
```
Comment (MikeSpreitzer) [inline docs/content/direct/get-started.md:39]: This was never needed; `kind` was always forceful about writing its context.

**Review by pdettori — COMMENTED** [review]
/lgtm

**Pull request merged**